media-gfx/graphviz/patches/graphviz-2.49.1.patchset

```diff
@@ -0,0 +1,95 @@
+From 981c5c5471e1611fc852084f2283f386579428a7 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Wed, 29 Sep 2021 15:26:33 +0000
+Subject: set Haiku font directories
+
+
+diff --git a/configure.ac b/configure.ac
+index fe3f2be..e706ee0 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -144,6 +144,7 @@ UWIN=no
+ CYGWIN=no
+ MINGW32=no
+ DARWIN9=no
++HAIKU=no
+ case "${host_os}" in
+   *uwin* )
+ 	UWIN=yes
+@@ -185,6 +186,17 @@ case "${host_os}" in
+ 		;;
+         esac
+ 	;;
++	*haiku* )
++	HAIKU=yes
++	HAIKU_SYSTEM_FONTPATH=`finddir B_SYSTEM_FONTS_DIRECTORY`
++	HAIKU_SYSTEM_NONPACKAGED_FONTPATH=`finddir B_SYSTEM_NONPACKAGED_FONTS_DIRECTORY`
++	HAIKU_USER_FONTPATH=`finddir B_USER_FONTS_DIRECTORY`
++	HAIKU_USER_NONPACKAGED_FONTPATH=`finddir B_USER_NONPACKAGED_FONTS_DIRECTORY`
++	DEFAULT_FONTPATH="$HAIKU_SYSTEM_FONTPATH:$HAIKU_SYSTEM_FONTPATH/ttfonts:$HAIKU_SYSTEM_FONTPATH/psfonts:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH/ttfonts:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH/psfonts:$HAIKU_USER_FONTPATH:$HAIKU_USER_FONTPATH/ttfonts:$HAIKU_USER_FONTPATH/psfonts:$HAIKU_USER_NONPACKAGED_FONTPATH:$HAIKU_USER_NONPACKAGED_FONTPATH/ttfonts:$HAIKU_USER_NONPACKAGED_FONTPATH/psfonts"
```

Comment:
could be split up (untested patch)
```suggestion
+	DEFAULT_FONTPATH="$HAIKU_SYSTEM_FONTPATH:$HAIKU_SYSTEM_FONTPATH/ttfonts:$HAIKU_SYSTEM_FONTPATH/psfonts"
+	DEFAULT_FONTPATH="$DEFAULT_FONTPATH:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH/ttfonts:$HAIKU_SYSTEM_NONPACKAGED_FONTPATH/psfonts"
+	DEFAULT_FONTPATH="$DEFAULT_FONTPATH:$HAIKU_USER_FONTPATH:$HAIKU_USER_FONTPATH/ttfonts:$HAIKU_USER_FONTPATH/psfonts"
+	DEFAULT_FONTPATH="$DEFAULT_FONTPATH:$HAIKU_USER_NONPACKAGED_FONTPATH:$HAIKU_USER_NONPACKAGED_FONTPATH/ttfonts:$HAIKU_USER_NONPACKAGED_FONTPATH/psfonts"
```

Reply:
Was thinking about that too, wasn't sure how to though

Reply:
That could work, just need to re-create the patchset for it then

Reply:
patchset updated, please check (build is ok on 32bit)